File: DomainModels/PropertyType.fs
namespace DomainModels

open System.ComponentModel.DataAnnotations

type PropertyTypeEnum =
    | Integer = 0
    | Decimal = 1
    | Option = 2
    
[<CLIMutable>]
type PropertyType =
    {
        [<Key>]
        Id:int
        Name:string
    }

File: WebShop_NULL/Models/DomainModels/PropertyType.cs
using System.ComponentModel.DataAnnotations;

namespace WebShop_NULL.Models.DomainModels
{
    public enum PropertyTypeEnum
    {
        Integer = 0,
        Decimal = 1,
        Option = 2,
        Nominal = 3
    }
    
    public class PropertyType
    {
        [Key]
        public int Id { get; set; }

        public string Name { get; set; }
    }
}